src/NzbDrone.Core/DecisionEngine/DownloadDecisionComparer.cs

```diff
@@ -28,6 +28,7 @@ public int Compare(DownloadDecision x, DownloadDecision y)
         {
             var comparers = new List<CompareDelegate>
             {
+                CompareSearchPriority,
```

Comment:
Indexer priority is not more important than quality, it should be just after protocol I think.